### PR TITLE
Update lutron_caseta.markdown

### PR DESCRIPTION
**Pull request by djj211**
**Description:**
Updated for new config point to work with my new pylutron_caseta mods and also support for the new caseta fan.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):**
https://github.com/home-assistant/home-assistant/pull/29033

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9780"></a>

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9780"></a>

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9780"></a>

### CODE REVIEW

File: source/_components/lutron_caseta.markdown [hunk 1]
```diff
@@ -14,13 +14,15 @@ ha_category:
...
   - Switch
+  - Fan
 ha_release: 0.41
 ha_iot_class: Local Polling
 redirect_from:
   - /components/cover.lutron_caseta/
   - /components/light.lutron_caseta/
   - /components/scene.lutron_caseta/
   - /components/switch.lutron_caseta/
+  - /components/fan.lutron_caseta/
```
Comment (klaasnicolaas) [inline source/_components/lutron_caseta.markdown]: Please remove line 25

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/lutron_caseta.markdown [hunk 2]
```diff
@@ -113,3 +119,11 @@ For dimmable lights including wall and plug-in dimmers, see the light section on
 For more information on working with switches in Home Assistant, see the [Switches component](/components/switch/).
 
 Available services: `switch.turn_on` and `switch.turn_off`.
+
+## {% linkable_title Fan %}
```
Comment (klaasnicolaas) [inline source/_components/lutron_caseta.markdown]:
```suggestion
## Fan
```

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Comment by MartinHjelmare** [comment]
Please rebase on latest next branch.

**Review by frenck — APPROVED** [review]
Rebased the PR and updated it during rebase to match the latest standards.

Thanks for the PR! 👍

**Pull request merged**